CMakeLists.txt

```
@@ -63,6 +63,7 @@ option(IREE_BUILD_PYTHON_BINDINGS "Builds the IREE python bindings" OFF)
 option(IREE_BUILD_EXPERIMENTAL_REMOTING "Builds experimental remoting support." OFF)
 option(IREE_BUILD_EXPERIMENTAL_WEB_SAMPLES "Builds experimental web samples." OFF)
 option(IREE_HAL_DRIVER_EXPERIMENTAL_ROCM "Builds the experimental ROCm Backend." OFF)
+option(IREE_BUILD_TORCH_MLIR_DIALECTS "Builds the experimental torch-mlir-dialects." ON)
```

"experimental" in the option description is at odds with being enabled by default, and I think we can clarify that this enables the TMTensor / Torch-MLIR integration (and not just the incidental torch-mlir-dialects dep)

```suggestion
option(IREE_BUILD_TORCH_MLIR_SUPPORT "Builds support for compiling torch-mlir programs." ON)
```

If we had a similar option for TOSA and MHLO we'd want the names to line up, but we can bikeshed on that later :)

---

Note that you'll also need to update all references to use the new option name. You local CMake cache will continue to remember the old value as long as you don't clear it.

---

sry, some how I missed the option name change.